src/common/snippets/src/pass/collapse_subgraph.cpp

```diff
@@ -58,7 +64,8 @@ auto is_supported_op(const std::shared_ptr<const Node> &n) -> bool {
                 // todo: {0, 2, 1, 3} Transpose should also be tokenized to support MHA pattern.
                 //  It's disabled because {0, 2, 1, 3} Transposes to be fused with MatMuls, that are
                 //  not tokenized yet. Enable tokenization of {0, 2, 1, 3} after ticket 95631 is implemented.
-                return TransposeDecomposition::supported_cases.count(order_value) != 0;
+                return TransposeDecomposition::supported_cases.count(order_value) != 0 ||
+                        order_value == std::vector<int>{0, 2, 1, 3};
```

Comment:
maybe replace on `FuseTransposeBrgemm::supported_cases`?

Reply:
Exactly, now we can do it) so let's do it